test/torch_np/numpy_tests/lib/test_arraysetops.py

```diff
@@ -217,7 +231,7 @@ def _isin_slow(a, b):
         isin_slow = np.vectorize(_isin_slow, otypes=[bool], excluded={1})
 
         def assert_isin_equal(a, b):
-            x = isin(a, b, kind=kind)
+            x = np.isin(a, b, kind=kind)
```

Comment:
what happened here before?

Reply:
The original numpy tests had `from numpy import isin`; when porting this we skipped/xfailed all tests which required it, and it never ran. We just never noticed that the xfail reason was an ImportError (or it was masked by other ImportErrors).

Now with a conditional import, we either need to add `If TEST_WITH_TORCHDYNAMO: from numpy import isin`, or do this line.

So in this PR, this will graph break (because we do not implement `torch._numpy.isin`).

Reply:
thank you for the explanation!